src/popup/fxa.js

```diff
@@ -64,3 +91,11 @@ function openSyncPreferences() {
   browser.fxa.openSyncPreferences();
   window.close();
 }
+
+function sendTelemetry(interactionType, elapsedTime) {
+  feature.sendTelemetry({
```

Comment:
And, of course, here is where `feature` is undefined. Can we bubble events up from this popup document to the parent feature class? If not is there some other way we can communicate between the two.

Worst case we can do the extra work to get all the state inside the popup js, it just seems like a shame to duplicate the effort is all.